source/templates.tex

```diff
@@ -1204,8 +1204,7 @@
 of the type of the \grammarterm{template-parameter}.
 \begin{note}
 If the \grammarterm{template-argument}
-represents a set of overloaded functions
-(or a pointer or member pointer to such),
+represents an overload set (or a pointer or member pointer to such),
```

Comment:
What is a pointer to an overload set? That doesn't seem to fit the new (or old) terminology very well.

Reply:
I'd be slightly more comfortable with "the address of an overload set", as it suggests use of the `&` operator rather than a specific pointer value. (And maybe "is" rather than "represents", since "represents the address of" is again a specifically defined property of real pointer values, rather than a syntactic property of an expression.)

Reply:
Fixed.